packages/contracts/smt-checker/smt/update.ts

```diff
@@ -0,0 +1,68 @@
+import fs from "fs";
+import path from "path";
+
+const updateFoundryToml = async () => {
+  const currentDirectory = process.cwd();
+  const foundryTomlPath = path.join(currentDirectory, "foundry.toml");
+
+  try {
+    // Read the contents of foundry.toml
+    let foundryTomlContent = fs.readFileSync(foundryTomlPath, "utf-8");
+
+    // Check if the model_checker section exists in foundry.toml
+    const regex = /\[profile\.default\.model_checker\]/;
+    const match = regex.exec(foundryTomlContent);
+
+    if (match) {
+      console.log("model_checker section already exists in foundry.toml.");
+      return;
+    }
+
+    // Generate the model_checker section content
+    const modelCheckerSectionContent = generateModelCheckerSectionContent();
+
+    // Update foundry.toml with the model_checker section
+    foundryTomlContent = updateFoundryTomlContent(foundryTomlContent, modelCheckerSectionContent);
+
+    // Write the updated content back to foundry.toml
+    fs.writeFileSync(foundryTomlPath, foundryTomlContent);
+
+    console.log("foundry.toml updated successfully.");
+  } catch (error) {
+    console.error("Error updating foundry.toml:", error);
+  }
+};
+
+const generateModelCheckerSectionContent = (): string => {
+  const modelCheckerSectionContent = `
+[profile.SMT.model_checker]
+contracts = { }
+engine = 'chc'
+solvers = ['z3']
+show_unproved = true
+timeout = 0
+targets = [
+  'assert',
+  'constantCondition',
+  'divByZero',
+  'outOfBounds',
+  'overflow',
+  'popEmptyArray',
```

Comment:
Could you also add `underflow` and `balance` here?

Reply:
Thanks for the info on the [security epic](https://github.com/ubiquity/ubiquity-dollar/issues/639#issuecomment-1554194502) link

Pushed in https://github.com/ubiquity/ubiquity-dollar/pull/702/commits/fe96d5f3a3ccebb1646d24025b5c7ff81d15be41